pkg/controller/controllerutil/controllerutil.go

```diff
@@ -119,6 +119,36 @@ func SetOwnerReference(owner, object metav1.Object, scheme *runtime.Scheme) erro
 
 }
 
+// HasExistingReference is a helper method to validate if existing object has provided owner as reference
+// It prevent objects from unnecessary updates in case of the owner has not changed
+func HasExistingReference(owner, object metav1.Object, scheme *runtime.Scheme) (bool, error) {
```

Comment:
```suggestion
// IsOwner is a helper method to validate if existing object has provided owner as reference
// It prevent objects from unnecessary updates in case of the owner has not changed
func IsOwner(owner, object Object, scheme *runtime.Scheme) (bool, error) {
```

Reply:
Fixed, thanks @vincepri